app/presenters/combined_report_presenter.rb

```diff
@@ -0,0 +1,27 @@
+class CombinedReportPresenter < ReportPresenter
+  def initialize(schemes: [],
+                 report_form: ReportForm.new(from_date: 14.months.ago, to_date: Date.current))
+    self.schemes = schemes
+    self.report_form = report_form
+  end
+
+  def defects
+    @defects ||= Defect.for_scheme(schemes.pluck(:id))
+                       .where(
+                         'created_at >= ? and created_at <= ?',
+                         report_form.from_date.beginning_of_day, report_form.to_date.end_of_day
+                       )
+  end
+
+  def defects_by_priority(priority:)
```

Comment:
I think `defects_by_priority_name(priority_name:)` might be more expressive here

Reply:
Both `CombinedReportPresenter` and `SeriesReportPresenter` share some common methods. Some of those methods call out to `defects_by_priority`. Renaming this would break that clean pattern.

If you don't mind I would like to keep this as is, for the greater good.

Reply:
Ah, OK. No worries 👍